packages/core/src/types.ts

```diff
@@ -1799,7 +1799,7 @@ export interface Behavior<TEvent extends EventObject, TEmitted = any> {
 }
 
 export type EmittedFrom<T> = ReturnTypeOrValue<T> extends infer R
-  ? R extends ActorRef<infer _, infer TEmitted>
+  ? R extends ActorRef<any, infer TEmitted>
```

Comment:
This is somewhat weird because my intuition was that it should not work when `R` has `TEvent` configured as `never`. 

And indeed it shouldn't work as per [this playground](https://www.typescriptlang.org/play?ts=4.6.2#code/JYOwLgpgTgZghgYwgAgIILAeygJQjAHgBUBRANwnGQgA9IQATAZ2QG9kwBPABwgC5kTMFFABzZAF8AfGwBQyQZQYCAFBDIDSFcAEpkAXhllMwBgG5ZE2bID0N5MDAOWtXhggMOACzhOwX4BYAd0wQAHInACMUOCYmYFEQOEiAGwhZLl5kIgghAEYDNAxsPEIQdWgZWnpmIqxcfAI4EE4ZAH5kAoEABlt7BQA9NutMlABlJWhickonaqUWdlGBIREQcWlCtW0wTRnwAG0InggwgF1kAB9s-bA9Q2RjUwtZUEhYRBR0etKAJmmdtQ6As2BwTithGJJDJWPJFIwBBNGFMtLMpJZrHZkM1PP5As5BGBgCkUsgQJgnLF4olkmkMidsrkwL9Ct8So1yhQoFVgYwWGyGoRmq1kB0usheljBm0gA). So it got me thinking... and I've realized that those becomes assignable when we turn off the `strictFunctionTypes` (see [here](https://www.typescriptlang.org/play?strictFunctionTypes=false&ts=4.6.2#code/JYOwLgpgTgZghgYwgAgIILAeygJQjAHgBUBRANwnGQgA9IQATAZ2QG9kwBPABwgC5kTMFFABzZAF8AfGwBQyQZQYCAFBDIDSFcAEpkAXhllMwBgG5ZE2bID0N5MDAOWtXhggMOACzhOwX4BYAd0wQAHInACMUOCYmYFEQOEiAGwhZLl5kIgghAEYDNAxsPEIQdWgZWnpmIqxcfAI4EE4ZAH5kAoEABlt7BQA9NutMlABlJWhickonaqUWdlGBIREQcWlCtW0wTRnwAG0InggwgF1kAB9s-bA9Q2RjUwtZUEhYRBR0etKAJmmdtQ6As2BwTithGJJDJWPJFIwBBNGFMtLMpJZrHZkM1PP5As5BGBgCkUsgQJgnLF4olkmkMidsrkwL9Ct8So1yhQoFVgYwWGyGoRmq1kB0usheljBsMgA)). 

The weird thing is that... our tests should be type-checked using strict mode but apparently this doesn't quite work 🤷‍♂️ 

Gonna fix this up in a sec.